src/Yoti.Auth/DataObjects/ProfileDO.cs

```diff
@@ -2,8 +2,8 @@
 {
     internal class ProfileDO
     {
-        public string session_data { get; set; }
+        public string SessionData { get; set; }
```

Comment:
We want these object to be immutable when returned to SDK clients.  We need to look at the use of _Set_ property.  Maybe use _internal_ or something, rather than _public_, if we really need to use _Set_ at all

Reply:
Since the `ProfileDO` class is internal, everything within it can at most be internal (and not public). 

The convention is to use public for everything within a class, as this just signifies to observe the top level accessibility defined (which is `internal` here).